vars/withGithubCheck.groovy

```diff
@@ -34,16 +34,7 @@ def call(Map args = [:], Closure body) {
   def org = args.get('org', env.ORG_NAME)
   def repository = args.get('repository', env.REPO_NAME)
   def commitId = args.get('commitId', env.GIT_BASE_COMMIT)
-  def redirect = args.get('tab', 'pipeline')
-  def isBo = args.get('isBlueOcean', false)
-
-  if (!redirect.startsWith('http')) {
-    if (isBo) {
-      redirect = getBlueoceanTabURL(redirect)
-    } else {
-      redirect = getTraditionalPageURL(redirect)
-    }
-  }
+  def redirect = detailsURL(tab: args.get('tab', 'pipeline'), isBlueOcean: args.get('isBlueOcean', false))
```

Comment:
I like having the input parameters list at the beginning of the method, so that it's possible to understand the expected params in the args map. Putting them here could hide them from an ocasional reader